benchmark/startup.md

```diff
@@ -44,10 +44,9 @@ print the `-version` message)
 * WAMR fast-jit seems lightweight for a JIT-based runtime as it's
   advertized.
   It also uses a lazy compilation strategy by default.
-  Unlike wasm3, it doesn't defer the validation though. Disabling
-  the lazy compilation (`-DWAMR_BUILD_LAZY_JIT=0`) doesn't make a much
-  difference as I expected. I'm not sure why.
-  Probably it's because of [a naive locking](https://github.com/bytecodealliance/wasm-micro-runtime/issues/2499).
+  Unlike wasm3 and wasmi, it doesn't defer the validation though.
```

Comment:
This can be emulated with Wasmi, too via `--compilation-mode lazy-translation`. Though I generally wouldn't recommend using that since it is just roughly 50% faster than eager translation but has all the downsides of lazy translation.

Reply:
ok. i didn't know the option.
personally i like lazy-translation mode because it wouldn't change visible behaviors from eager.

Reply:
Yes, I also like it a lot which is why I introduced both lazy variants. But in practical use cases it often happens that inputs are pre-validated so the `lazy` option is as safe and much more efficient than the `lazy-translation` option.